.github/workflows/build.yml

```diff
@@ -36,6 +36,12 @@ jobs:
       image: ${{ matrix.container-image }}
 
     steps:
+      - name: GitHub Actions dependencies (openSUSE only)
```

Comment:
You could just move `- uses: actions/checkout@v3` below the dependency installation. Then there is only one step for deps and it does not need to be temporary.

BTW, you need only tar and gzip here.

Reply:
I'm deliberately keeping it separate since I expect it to be removed once the containers are updated.

Reply:
What's so bad about just moving one step?

Reply:
I guess because then it can be removed just from here.